libwacom/libwacom-database.c

```diff
@@ -1093,12 +1250,11 @@ load_stylus_files(WacomDeviceDatabase *db, const char *datadir)
 
 	while ((file = readdir(dir))) {
 		char *path;
-
 		if (!is_stylus_file(file))
 			continue;
 
 		path = g_build_filename (datadir, file->d_name, NULL);
-		libwacom_parse_stylus_keyfile(db, path);
+		libwacom_parse_stylus_keyfile(db, path, alias);
```

Comment:
heh, see my comment above for immediately readable by naming to `handle_aliases` because I looked at this and was wondering why we're passing an alias into the keyfile parser